### ISSUE TITLE
New signatures on COSMIC website

### ISSUE THREAD

**Issue by anishaluthra** [AlexandrovLab/SigProfilerSingleSample#5]
Hi,

I noticed that there were new signatures that were defined and put on the COSMIC website. I want to compute scores for one of the new signatures (SBS88), but I have exome data only. There is a signature definition that can be downloaded but I believe this is the genome signature definition; however, I believe that the exome signature definitions are different from the genome signature definitions (as the exome and genome signature text files are different). Was wondering how the genome signatures definitions are normalized to convert to exome signature definitions?

In other words, if I wanted to use SigProfilerSingleSample to find scores for SBS88 with my exome data, would I be able to use the definition that I can download from the COSMIC website or would it require some normalization?

Thanks in advance for your help!

**Comment by mishugeb** [comment]
@marcos-diazg, could you please help me to answer the question? I am a little confused.

**Comment by mishugeb** [comment]
There is no way to use genome signatures to fit in exome data. Exome signatures have to be extracted from the exome catalog.

**Comment by anishaluthra** [comment]
Okay, so I wouldn't be able to calculate scores for SBS88, given what's on the COSMIC website, for exome data?

**Comment by anishaluthra** [comment]
Will you be updating the exome and genome signature files with the new signatures from release 3.1?

**Comment by rptashkin** [comment]
would also be interested in COSMIC v3.1 signatures

**Comment by marcos-diazg** [comment]
We are currently working on a major update of the tool that will include the option to choose among the different versions of COSMIC reference mutational signatures, including current version 3.2 (released on March 2021). We hope to have a new version in the next few months. Thanks a lot for your suggestions and interest!
